source/Calamari.AzureWebApp/NetFxWebDeploymentExecutor.cs

```diff
@@ -0,0 +1,181 @@
+﻿#if NETFRAMEWORK
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Calamari.Azure.AppServices;
+using Calamari.AzureWebApp.Integration.Websites.Publishing;
+using Calamari.AzureWebApp.Util;
+using Calamari.Common.Commands;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+using Microsoft.Web.Deployment;
+
+namespace Calamari.AzureWebApp
+{
+    public class NetFxWebDeploymentExecutor : IWebDeploymentExecutor
```

Comment:
This uses the `Microsoft.Web.Deployment` libraries, which there aren't NetCore versions of